image_provisioner/playbooks/roles/cri-o/templates/crio.conf.j2

```diff
@@ -0,0 +1,123 @@
+# The "crio" table contains all of the server options.
```

Comment:
Having the whole config file in our git is going to get very ugly.  Suggest thinking this through more...there will be no way to keep up with the upstream config file options.  Perhaps only include things that we modify and use lineinfile ansible module to juggle it?